cmd/cmd.go

```diff
@@ -20,6 +20,9 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
+// Version is the current semvar or "development" if not production
+var Version = "development"
```

Comment:
any `go install` version will just be development, unless setting ldflags manually.

Reply:
Cool! I'm partial to `govvv` as well here... it's nice to see how far off the latest mater commit you are, and if the branch is "dirty" or not. What you have here could be used in conjunction w/ that, i.e., using `make` would properly setup `govvv`, but just a normal `go install` would use this default `Version` tag. Thoughts?

Reply:
yep, that's easy